docs/grammar/BallerinaParser.g4

```diff
@@ -528,7 +528,10 @@ fieldDefinition
     ;
 
 simpleLiteral
-    :   (SUB)? IntegerLiteral
```

Comment:
Changing this seems not correct. This should be handled by the parser listener.  @sameera-jayasoma WDTY ?

Reply:
We can do the parsing at ballerina level. But it needs costly string comparisons and there is no way to prioritize the decimal values. That why it moved to the parser level.

Reply:
That is an implementation detail. This makes grammar complicated for external users IMO.